src/components/scene/wakelock.js

```diff
@@ -0,0 +1,15 @@
+var register = require('../../core/component').registerComponent;
+var Wakelock = require('../../../vendor/wakelock/wakelock');
```

Comment:
Same for this. Not sure it should be a component. This should not be optional or configurable.